randovania/layout/preset_migration.py

```diff
@@ -1128,15 +1128,25 @@ def pop_from_base_config(field: str) -> None:
 
 
 def _migrate_v101(preset: dict, game: RandovaniaGame) -> None:
-    banned_pickups = ["Cannon Ball", "Unlimited Beam Ammo", "Unlimited Missiles", "Double Damage"]
-    pickup_config = preset["configuration"]["standard_pickup_configuration"]["pickups_state"]
     if game == RandovaniaGame.METROID_PRIME_ECHOES:
+        banned_pickups = ["Cannon Ball", "Unlimited Beam Ammo", "Unlimited Missiles", "Double Damage"]
+        pickup_config = preset["configuration"]["standard_pickup_configuration"]["pickups_state"]
         for pickup in banned_pickups:
             if pickup in pickup_config:
                 if "num_included_in_starting_pickups" in pickup_config[pickup]:
                     pickup_config[pickup]["num_included_in_starting_pickups"] = 0
 
 
+def _migrate_v102(preset: dict, game: RandovaniaGame) -> None:
```

Comment:
There's not really a reason for this migration. If something is wrong with the previous one, fix it since it hasn't hit stable yet.

Reply:
I guess that was done because I was asking what was wrong with the one before (but I didn't say it has to be a new one atm).
There is a corner case for me where a new one is better even if didn't hit stable yet. If it was corrupted by the migration before and anyone saved it that way in the meantime, the corrupted one will also be used in stable.